configs/benchmark/nav/imagenav/imagenav_hm3d_v3_challenge.yaml

```diff
@@ -0,0 +1,34 @@
+# @package _global_
+
+defaults:
+  - /habitat: habitat_config_base
+  - /habitat/task: imagenav_v3_challenge
+  - /habitat/simulator/agents@habitat.simulator.agents.main_agent: rgbd_agent
+  - /habitat/dataset/imagenav: hm3d_v3_challenge
+  - _self_
+
+habitat:
+  environment:
+    max_episode_seconds: 1000
+  simulator:
+    action_space_config: velocitycontrol
+    agents:
+      main_agent:
+        sim_sensors:
+          rgb_sensor:
+            width: 480
+            height: 640
+            hfov: 42
+            position: [0, 1.31, 0]
+          depth_sensor:
+            width: 480
+            height: 640
+            hfov: 58
```

Comment:
This should be 42.
@ykarmesh, mind fixing?

Reply:
Oh IDK how I never noticed this